proxyd/backend.go

```diff
@@ -711,11 +711,12 @@ func sortBatchRPCResponse(req []*RPCReq, res []*RPCRes) {
 }
 
 type BackendGroup struct {
-	Name             string
-	Backends         []*Backend
-	WeightedRouting  bool
-	Consensus        *ConsensusPoller
-	FallbackBackends map[string]bool
+	Name              string
+	Backends          []*Backend
+	WeightedRouting   bool
+	Consensus         *ConsensusPoller
+	FallbackBackends  map[string]bool
+	MulticallBackends map[string]bool
```

Comment:
how this will be configured?

I thought we would just have a new config option in the backend group with three potential values. This should replace the `consensus_aware` boolean to something like `routing_strategy` which can be `fallback`, `consensus` or `fanout`. 

Configuring a bg as a fallback should just use the same list of backends we already have.